### PR TITLE
Update prediction.md reference to Pinpoint

### PR DESCRIPTION
**Pull request by jodion**
Changing the reference of Pinpoint in the manual setup section to Cognito Federated Identities.

*Issue #, if available:*

*Description of changes:*
There was a typo referring to Pinpoint in the documentation. I believe it should be about Cognito Federated Identities.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: js/predictions.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Amplify.addPluggable(new AmazonAIPredictionsProvider());
 
 #### Manual Setup
 
-The manual setup enables you to use your existing Amazon Pinpoint resource in your app.
+The manual setup enables you to use your existing Amazon Cognito Identity Pool resource in your app.
```
Comment (elorzafe) [inline js/predictions.md]:
```suggestion
The manual setup enables you to use your existing Amazon AI and ML resources in your app.
```

Comment (elorzafe) [reply js/predictions.md]: Thanks for pointing this out, the idea is that if you have already configured permissions (IAM roles and policies on your identity pool to have access to AI services) you can manually configure those on the library

Comment (jodion) [reply js/predictions.md]: That makes sense. I was using Cognito as that was the only configurable parameter in the code section below, but if you are using Rekognition, you may want to point to your own face library and Cognito would be used from a different category. So what you wrote is better.

Comment (elorzafe) [reply js/predictions.md]: @jodion can you commit the suggestion so I can merge the pr. Thanks!

**Pull request merged**